src/Orchard.Cms.Web/Modules/Orchard.Setup/Views/Setup/Index.cshtml

```diff
@@ -133,3 +157,22 @@
         </div>
     </fieldset>
 </form>
+<script src="~/Orchard.Setup/Scripts/setup.min.js" type="text/javascript"></script>
+<script type="text/javascript">
+    //<![CDATA[
+    $(function(){
+        $('#Password').bootstrapStrength({
+            minLength: @(options.Password.RequiredLength),
+            upperCase: @(Convert.ToInt16(options.Password.RequireUppercase)),
```

Comment:
use `options.Password.RequireUppercase ? 1 : 0` instead of `Convert`, otherwise I need to think about what it would mean